### PR TITLE
add_rishabh__

### PR DESCRIPTION
**Pull request by rishistyping**
- [ ] Included a Preview link and screenshot showing after and before the changes.
- [ ] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](url)

- Fixes #<Issue Number>

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by 0x48piraj** [comment]
Can you provide a preview link?

File: _config.yml [hunk 1]
```diff
@@ -6,3 +6,5 @@ sass:
 
 plugins:
     - jekyll-gzip
+
+theme: jekyll-theme-cayman
```
Comment (0x48piraj) [inline _config.yml]: It's not needed.

**Comment by EXYNOS-999** [comment]
Screenshot from 2020-01-02 17-40-04

**Review by 0x48piraj — COMMENTED** [review]
Why added the cayman theme?

**Comment by EXYNOS-999** [comment]
Which one do you guys use?

**Comment by 0x48piraj** [comment]
None, we have built it from scratch.

**Comment by EXYNOS-999** [comment]
Just to test it out, again I'm a guy with 0 experience with static webpage generators.

**Comment by 0x48piraj** [comment]
No problem, great work, thanks!

**Pull request merged**